aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1647,6 +1690,110 @@ Tensor _cholesky_helper_cuda(const Tensor& self, bool upper) {
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ cholesky_inverse ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/*
+Copies the lower (or upper) triangle of the square matrix to the other half and conjugates it.
+This operation is performed in-place.
+*/
+template <typename scalar_t, typename IndexType>
+#ifdef __HIP_PLATFORM_HCC__
+C10_LAUNCH_BOUNDS_1(512)
+#endif
+__global__
+void reflect_conj_tri_single_kernel(
```

Comment:
Add a comment explaining this function and how it's intended to be used

Reply:
I removed it, it's not needed anymore.